python/cudf/cudf/_lib/column.pyx

```diff
@@ -388,8 +392,10 @@ cdef class Column:
         if has_nulls:
             mask = DeviceBuffer.c_from_unique_ptr(move(contents.null_mask))
             mask = Buffer(mask)
+            null_count = c_col.get()[0].null_count()
         else:
             mask = None
+            null_count = None
```

Comment:
This should be 0 instead of None since there's no mask, right?

Reply:
Correct

Reply:
Fixed